site/layouts/partials/header.html

```diff
@@ -1,7 +1,7 @@
 <header class="site-header">
     <div class="wrapper flex jc-sb ai-c">
     <div class="flex">
-        <a href="/" class="logo">
+        <a href="/" aria-label="return to landing page" class="logo">
```

Comment:
it's possible that this wording is a bit off, so no problem updating as necessary in this PR.